tests/testthat/test_semanticPage.R

```diff
@@ -0,0 +1,16 @@
+context("semanticPage")
+
+test_that("test get_cdn_path return default path", {
+  # When
+  options("shiny.custom.semantic.cdn" = NULL)
+  # Then
+  expect_equal(get_cdn_path(), "https://d335w9rbwpvuxm.cloudfront.net")
+})
+
+test_that("test get_cdn_path return path", {
+  # When
+  options("shiny.custom.semantic.cdn" = "shiny.semantic")
```

Comment:
nit: I think the value in the test is misleading, instead I suggest using some fake url for custom cdn path

Reply:
`shiny.semantic` is valid URL when you would like to serve files locally from shiny. In shiny 1.3 the speed of hosting static files was improved dramatically, so we can even think about move semantic dependencies to be part of the code.